README.md

```diff
@@ -42,12 +42,16 @@ Complete the following steps to setup a local Edge Server.
    ```
    docker run hello-world
    ```
-3. Open the terminal and pull the latest edge-server image then run the Edge Server container with the following commands:
+3. Open the terminal and pull the latest edge-server image, then run the Edge Server container with the following commands.
    ```
-   docker pull datafi/es:latest
-   docker run --rm -p 80:80 -p 443:443 datafi/es:latest
+   docker pull docker://datafi/es:latest
```

Comment:
this won't work in default mac docker installations. It says `invalid reference format`

```
➜  ~  docker pull docker://datafi/es:latest
invalid reference format
➜  ~  docker pull datafi/es:latest         
latest: Pulling from datafi/es
Digest: sha256:02da8de4651f23f6c8e4c75c73d4d88c497ba228ec106cfe93f4da9c323ba3ef
Status: Image is up to date for datafi/es:latest
docker.io/datafi/es:latest
```

Reply:
I assume this is specific to Poddman. may be we can add another section in the footer, for those who use Podman and explain details.